realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -207,6 +222,27 @@ public void renameColumn(long columnIndex, String newName) {
 
     protected native void nativeRenameColumn(long nativeTablePtr, long columnIndex, String name);
 
+    /**
+     * Is a column nullable?
+     *
+     * @param columnIndex
+     * @return true if column is nullable, false otherwise.
+     */
+    public boolean isColumnNullable(long columnIndex) {
+        return nativeIsColumnNullable(nativePtr, columnIndex);
+    }
+
+    protected native boolean nativeIsColumnNullable(long nativePtr, long columnIndex);
```

Comment:
We started grouping these at the bottom of the file. If this file has not yet been converted, keep it as it is as there is no reason to add more "clutter" to this PR.